suite/spdz/src/main/java/dk/alexandra/fresco/suite/spdz/storage/SpdzMascotDataSupplier.java

```diff
@@ -0,0 +1,161 @@
+package dk.alexandra.fresco.suite.spdz.storage;
```

Comment:
As mentioned in the spdz POM file comment, this class should be located within the mascot project.